ps2.h

```diff
@@ -355,6 +352,15 @@ const uint8_t PS2_REG_SCANCODES[] PROGMEM = {
   0, 0, 118
 };
 
+const uint8_t ps2ext_to_keycode_input[] PROGMEM = {
```

Comment:
These two arrays have tight coupling between them. Would it be better to make them uint8_t[][2] so that the input and output codes go together as pairs explicitly?

Reply:
That would for sure improve readability, but may affect code size and speed. I will check if the tables can be defined in a more readable way, while also sufficiently small and fast.

Reply:
Resolved, as I have reverted this part of the pull request.